app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title
+    base_title = t("base_title")
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
+    end
```

Comment:
dùng toán tử 3 ngôi nhé, phân biệt thêm giúp anh: nil?, empty?, present?, blank?

Reply:
nil? hàm của Ruby kiểm tra object có nil hay không
empty? hàm của Ruby kiểm tra String, Hash, Array, trả về true khi rỗng
blank? hàm thêm của Rails trả về true khi object false, empty hoặc chỉ chứa khoảng trắng. Ngược lại trả về false
present? hàm thêm của Rails và trả về kết quả ngược lại với blank.